rest/route/patch_test.go

```diff
@@ -521,13 +567,13 @@ func (s *PatchesByUserSuite) TestEmptyTimeShouldSetNow() {
 
 func TestPatchRawHandler(t *testing.T) {
 	route := &patchRawHandler{
-		sc: &data.MockConnector{
-			URL: "https://evergreen.example.net",
-			MockPatchConnector: data.MockPatchConnector{
-				CachedRawPatches: map[string]string{
-					"":        "main diff",
-					"module1": "module1 diff",
-				},
+		sc: &data.DBConnector{
+			URL:              "https://evergreen.example.net",
+			DBPatchConnector: data.DBPatchConnector{
+				//CachedRawPatches: map[string]string{
```

Comment:
Should this be commented out?

Reply:
Removed, this was an old property on the mock.